### DISCUSSION TITLE
zwavejsui version for zwave 7.15.4 controller version

### DISCUSSION THREAD

**Discussion by priyanka043** [zwave-js/zwave-js-ui#3343] [Q&A]
we are having zwave ncp version 7.15.4 and currently we are using zwavejs ui version is 8.23.1 version and we are facing some inclusion and exclusion issues, we cannot update the ncp version as it is already at one of our client location, so we wanted to know which version of zwavejs ui we can use for 7.15.4 zwave controller.

**Comment by robertsLando** [comment]
cc @AlCalzone

**Comment by AlCalzone** [comment]
@priyanka043 Which kinds of issues are you experiencing? 
Please make a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs), loglevel `debug` and attach it here as a file (drag & drop into the text field).

**Comment by priyanka043** [comment]
[image: zwave_2.png]

[image: zwave_1.png]
[image: zwave_3.png]

**Reply by robertsLando** [reply]
@priyanka043 Images are not loaded

**Reply by AlCalzone** [reply]
You have to reply via the Github UI, not via E-Mail.

**Comment by priyanka043** [comment]
Hi PFA logs
![zwave_1](https://github.com/zwave-js/zwave-js-ui/assets/139616028/99707e95-7ed2-4948-840e-8609827b69a2)
![zwave_2](https://github.com/zwave-js/zwave-js-ui/assets/139616028/12401ebf-3533-4688-8230-5024a92aad32)
![zwave_3](https://github.com/zwave-js/zwave-js-ui/assets/139616028/788e2ac4-0084-4220-83cb-1795389324a6)

**Reply by AlCalzone** [reply]
Status `Failed` is returned by the controller when it had trouble communicating with the node you're trying to exclude or include. This can have several reasons, none of which are going to be solved by choosing a different version of Z-Wave JS:
- The controller is in a weird state and either needs to be soft-reset (restarted) or power-cycled (physically unplugged and re-plugged)
- The node to be included is in a weird state and needs to be power cycled
- There's a lot of noise on the network, causing RF frames to get lost. In the first log I see that the background noise level on channel 0 is 20 dBm higher than on the others - this could have something to do with it. Also you see that Node 24 is repeating commands. This could be the reason or a symptom of this.

In addition, all 700 series controller firmwares below 7.17.2 have serious bugs and/or some kind of communication issues. Personally, I wouldn't go higher either. Not sure how the 7.18.x behave (I got mixed reports) and 7.19.x is totally broken.
You may have to upgrade it after all - this can be done through Z-Wave JS UI.
